elastic/integrations
# Add event fields to log data streams

## What does this PR do?

This adds a variety of `event` fields to data streams where they were missing before. This fixes an issue where `event.code` was erroneously reported as `long`.

## Checklist



## How to test this PR locally

- Pull down and build
- Run `auth`, `system`, or `syslog`, check to see if `event.code` exists, and that the mapping is correct.
